plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -257,110 +256,114 @@ mfxStatus QSV_Encoder_Internal::InitParams(qsv_param_t *pParams,
 	m_mfxEncParams.mfx.FrameInfo.CropY = 0;
 	m_mfxEncParams.mfx.FrameInfo.CropW = pParams->nWidth;
 	m_mfxEncParams.mfx.FrameInfo.CropH = pParams->nHeight;
-	if (codec == QSV_CODEC_AV1)
-		m_mfxEncParams.mfx.GopRefDist = 1;
-	else
-		m_mfxEncParams.mfx.GopRefDist = pParams->nbFrames + 1;
-
-	if (codec == QSV_CODEC_HEVC)
-		m_mfxEncParams.mfx.LowPower = MFX_CODINGOPTION_OFF;
+	m_mfxEncParams.mfx.GopRefDist = pParams->nbFrames + 1;
 
+	mfxPlatform platform;
+	MFXVideoCORE_QueryPlatform(m_session, &platform);
 #if defined(_WIN32)
-	// TODO: Why isn't LowPower coding supported on VAAPI backend.
-	enum qsv_cpu_platform qsv_platform = qsv_get_cpu_platform();
-	if ((m_isDGPU || qsv_platform >= QSV_CPU_PLATFORM_ICL ||
-	     qsv_platform == QSV_CPU_PLATFORM_UNKNOWN) &&
-	    (pParams->nbFrames == 0) &&
-	    (m_ver.Major == 1 && m_ver.Minor >= 31)) {
+#pragma warning(push)
+#pragma warning(disable : 4996)
```

Comment:
Use `PRAGMA_` macros defines `libobs/util/c99defs.h` even if it's Windows-only.

Reply:
Fixed.